src/checks/checks.ts

```diff
@@ -139,6 +142,9 @@ export default class Checks {
         let inProgressQueuedWaiting = [checkStatus.IN_PROGRESS, checkStatus.QUEUED, checkStatus.WAITING]
         let anyInProgressQueuedWaiting = checks.filter(check => inProgressQueuedWaiting.includes(check.status));
         if (anyInProgressQueuedWaiting.length > 0) {
+            anyInProgressQueuedWaiting.forEach((check) => {
+                core.info(`Waiting for check completion of ${JSON.stringify(check.name)}`);
+            });
```

Comment:
I am not sure about this. I believe this would log every in progress, queued or waiting check on each iteration. The logs might get noisy very quickly if you have a lot of checks/iterations

Reply:
Yeah, this probably stemmed more from finding this useful during debugging Enterprise, but I can see how it's probably not necessary. I've removed this part.

Reply:
Apologies for the flurry of comments, but I actually made one more tweak regarding logging based on using it for a bit with the logging removed: I still actually found value in having these logs to better understand how `check_include` or `check_exclude` also behaved on each iteration. So I've restored this logging, but hidden it behind a new `verbose: true` option that the action accepts (but it defaults to `false`, so the existing logging behavior should be the same). Does that seem like a decent approach? Or if you'd still like to remove this optional logging, I can also do that.

Reply:
Having it as an option which defaults to false seems okay to me.